airbyte-integrations/connectors/source-file/source_file/spec.json

```diff
@@ -0,0 +1,140 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/file",
+
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "File Source Spec",
+    "type": "object",
+    "required": ["url", "storage"],
+
+    "properties": {
+      "format": {
+        "type": "string",
+        "enum": [
+          "csv",
+          "json",
+          "html",
+          "excel",
+          "feather",
+          "parquet",
+          "orc",
+          "pickle"
+        ],
+        "default": "csv"
+      },
+      "reader_options": {
+        "type": "string"
+      },
+
+      "storage": {
+        "type": "string",
+        "enum": [
```

Comment:
Shouldn't the enum be the name of the service?
```
HTTP
S3
GCS
...
```

Reply:
Yes, I wasn't sure how we were going to display things in the UI, I can update it to more general names